cmd/zetaclientd/start.go

```diff
@@ -13,6 +13,7 @@ import (
 	"syscall"
 	"time"
 
```

Comment:
The block `// start zeta supply checker` in this file can be removed, we will not implement a supply checker this way

Reply:
Ok, can be addressed in https://github.com/zeta-chain/node/issues/3112